src/packages/user/user/workspace/components/user-workspace-profile-settings/user-workspace-profile-settings.element.ts

```diff
@@ -0,0 +1,138 @@
+import { UMB_USER_WORKSPACE_CONTEXT } from '../../user-workspace.context.js';
+import { html, customElement, state, ifDefined, css } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UUISelectElement } from '@umbraco-cms/backoffice/external/uui';
+import { UserResponseModel } from '@umbraco-cms/backoffice/backend-api';
+import { UMB_AUTH, UmbLoggedInUser } from '@umbraco-cms/backoffice/auth';
+import { firstValueFrom } from '@umbraco-cms/backoffice/external/rxjs';
+import { umbExtensionsRegistry } from '@umbraco-cms/backoffice/extension-registry';
+
+@customElement('umb-user-workspace-profile-settings')
+export class UmbUserWorkspaceProfileSettingsElement extends UmbLitElement {
+	@state()
+	private _user?: UserResponseModel;
+
+	@state()
+	private _currentUser?: UmbLoggedInUser;
+
+	@state()
+	private languages: Array<{ name: string; value: string; selected: boolean }> = [];
+
+	#authContext?: typeof UMB_AUTH.TYPE;
+	#userWorkspaceContext?: typeof UMB_USER_WORKSPACE_CONTEXT.TYPE;
+
+	constructor() {
+		super();
+
+		this.consumeContext(UMB_AUTH, (instance) => {
+			this.#authContext = instance;
+			this.#observeCurrentUser();
+		});
+
+		this.consumeContext(UMB_USER_WORKSPACE_CONTEXT, (instance) => {
+			this.#userWorkspaceContext = instance;
+			this.observe(this.#userWorkspaceContext.data, (user) => (this._user = user));
+		});
+	}
+
+	#onLanguageChange(event: Event) {
+		const target = event.composedPath()[0] as UUISelectElement;
+
+		if (typeof target?.value === 'string') {
+			this.#userWorkspaceContext?.updateProperty('languageIsoCode', target.value);
+		}
+	}
+
+	#observeCurrentUser() {
+		if (!this.#authContext) return;
+		this.observe(this.#authContext.currentUser, async (currentUser) => {
```

Comment:
note about missing ControllerAlias, to make observe replace it self if this method is called again.